addons/l10n_ar/models/account_fiscal_position.py

```diff
@@ -13,31 +13,17 @@ class AccountFiscalPosition(models.Model):
 
     @api.model
     def _get_fiscal_position(self, partner, delivery=None):
-        """ Take into account the partner afip responsibility in order to auto-detect the fiscal position """
-        if self.env.company.country_id.code != "AR":
-            return super()._get_fiscal_position(partner, delivery=delivery)
-
-        # if no delivery use invoicing
-        if not delivery:
-            delivery = partner
-
-        # partner manually set fiscal position always win
-        manual_fiscal_position = delivery.property_account_position_id or partner.property_account_position_id
-        if manual_fiscal_position:
-            return manual_fiscal_position
+        company = self.env.company
+        if company.country_id.code == "AR":
```

Comment:
I tend to prefer that we stick with the existing guard-style condition.
```suggestion
        if self.env.company.country_id.code != "AR":
            return super()._get_fiscal_position(partner, delivery=delivery)
```